### PR TITLE
#5: :arrow_up: version updates

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**

### CODE REVIEW

File: habushu-mixology/pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,8 @@ license = "MIT License"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-krausening = "13"
+krausening = "16"
+cryptography = "^39.0.1"
```
Comment (kdendtler) [inline habushu-mixology/pyproject.toml:11]: Q/A: Do we want to consider using a specific version of cryptography?

**Review by kdendtler — APPROVED** [review]

File: habushu-mixology/pyproject.toml (hunk 1 above)
Comment (d-ryan-ashcraft) [reply habushu-mixology/pyproject.toml:11]: We could - but my understanding is the due to transitive dependency challenges within PIP/Python in general, using ranges for libraries like this is preferrable

Comment (ekoniec1) [reply habushu-mixology/pyproject.toml:11]: Long but good read on taking this approach a step forward and **not** capping the upper bounds of dependency versions unless needed: https://iscinumpy.dev/post/bound-version-constraints/

**Comment by d-ryan-ashcraft** [comment]
2.4.0 Release Notes updated to reflect changes.

**Pull request merged**